katdal/chunkstore_s3.py

```diff
@@ -57,6 +57,16 @@
 from .sensordata import to_str
 
 
+# Lifecycle policies unfortunately use XML encoding rather than JSON
+# Following path of least resistance we simply .format() this string
+# with the number of days for the expiry (and produced a sanitised
+# ID at the same time.
+_BASE_LIFECYCLE_POLICY = """<?xml version="1.0" encoding="UTF-8"?>
+<LifecycleConfiguration><Rule>
+<ID>katdal_expiry_{0}_days</ID><Prefix></Prefix><Status>Enabled</Status>
```

Comment:
Looking at https://docs.aws.amazon.com/AmazonS3/latest/dev/intro-lifecycle-rules.html, you also need to wrap `<Prefix>` in a `<Filter>` tag?

Reply:
I had an issue with doing that earlier, but it may have been conflated with some other issue. Let me test further...

Reply:
Seems to work - now including an empty <Filter></Filter> element as this seems valid and conveys our intent more clearly.